registration-processor-default.properties

```diff
@@ -985,11 +986,12 @@ mosip.role.registration.getPostsync=REGISTRATION_ADMIN,REGISTRATION_PROCESSOR,RE
 mosip.role.registration.getPostsyncv2=REGISTRATION_ADMIN,REGISTRATION_PROCESSOR,REGISTRATION_OFFICER,REGISTRATION_SUPERVISOR,RESIDENT
 auth.server.admin.allowed.audience=mosip-regproc-client,mosip-admin-client,mosip-resident-client,mosip-reg-client
 mosip.regproc.cbeff-validation.mandatory.modalities=Right,Left,Left RingFinger,Left LittleFinger,Right RingFinger,Left Thumb,Left IndexFinger,Right IndexFinger,Right LittleFinger,Right MiddleFinger,Left MiddleFinger,Right Thumb,EXCEPTION_PHOTO
-landing.zone.account.name=landing-zone
-landing.zone.type=DMZServer
+mosip.regproc.landing.zone.account.name=landing-zone
+mosip.regproc.landing.zone.type=DMZServer
```

Comment:
can we add some comments here about the rename here, which will help people when they are upgrading from 1.1.5.5 to 1.2.0.1.

Reply:
comment added.